test/activations.jl

```diff
@@ -131,38 +102,33 @@ end
     end
 end
 
-@testset "Test Integer64 and Integer32 inputs will force Float64 outputs" begin
-    test_value_int_input_forces_float64.(filter(x -> (x != relu && x != relu6 && x != hardtanh && x != trelu), ACTIVATION_FUNCTIONS))
+@testset "Integer inputs" begin
+    # These should work without error, for e.g. readme examples,
+    # but no serious use will involve integers, no need for performance.
+    @testset "$a" for a in ACTIVATION_FUNCTIONS
+        @test typeof(a(Int64(1))) <: Real
+        @test typeof(a(Int32(1))) <: Real
+    end
 
-    @testset "relu: " begin
-        # relu doesn't have to force floating point outputs
+    # The following ones can pass integers through. But it's not very important.
+    @testset "relu: Int -> Int" begin
```

Comment:
I simplified the integer tests. There's no real need to test forever that Int32 is type-stable; they should work with Int, but I can't imagine them being useful in anything performance-sensitive. 

Also, long long testset names cause things to wrap badly and become hard to read. Which was what motivated taking the axe to these.